### PR TITLE
Raees/test enrollment coupon

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@edx/testeng 
Please review. This PR is against https://github.com/edx/edx-e2e-tests/pull/171

**Comment by raeeschachar** [comment]
jenkins run bokchoy

**Comment by raeeschachar** [comment]
@benpatterson @estute @Muddasser @jzoldak 
Good to merge?

**Comment by raeeschachar** [comment]
jenkins run e2e

File: regression/pages/whitelabel/logout_page.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        self.q(
+            css='.user-menu>.btn.btn-default.dropdown-toggle.'
+            'hidden-xs.nav-button'
+        ).click()
+        self.wait_for_element_visibility(
+            '.dropdown-menu',
+            'wait for user dropdown to expand'
+        )
+        self.q(css='.nav-link[href="/logout/"]').click()
+        HomePage(self.browser).wait_for_page()
```
Comment (Muddasser) [inline regression/pages/whitelabel/logout_page.py]: Should it be outside the page object?

File: regression/tests/whitelabel/voucher_tests_base.py [hunk 2]
```diff
@@ -109,6 +127,36 @@ def redeem_single_course_discount_coupon(self, coupon_url):
         self.verify_course_info_on_coupon_redeem_page(redeem_coupon_page)
         redeem_coupon_page.click_checkout_button(self.course_id)
 
+    def redeem_single_course_enrollment_coupon(self, coupon_url, target_page):
+        """
+        Redeem single course enrollment coupon
+        Args
```
Comment (Muddasser) [inline regression/tests/whitelabel/voucher_tests_base.py]: Please add a little more description about what these arguments are.

**Comment by raeeschachar** [comment]
@benpatterson @jzoldak @estute 
Reminder on this :)

**Pull request merged**